pkg/controller/statefulset/stateful_set_control_test.go

```diff
@@ -2945,47 +2938,52 @@ func fakeResourceVersion(object interface{}) {
 
 func TestParallelScale(t *testing.T) {
 	for _, tc := range []struct {
-		desc            string
-		replicas        int32
-		desiredReplicas int32
+		desc                        string
+		replicas                    int32
+		desiredReplicas             int32
+		expectedMinParallelRequests int
```

Comment:
`expectedMinParallelRequests` is a new field? Why the original `1` is not enough?

Reply:
I have moved this refactoring to a new commit: https://github.com/kubernetes/kubernetes/pull/124624/commits/7b725f3bfcc0eb88a75f91f523e7b669213321ef.

This is to make the test more accurate as originally we expected only 2 parallel requests. But if we create a 1000 pods we should expect more, otherwise it would suggest a regression.